dffml/high_level.py

```diff
@@ -258,3 +258,61 @@ async def predict(
                 )
                 if update:
                     await sctx.update(record)
+
+
+async def load(source: BaseSource, *args: str) -> AsyncIterator[Record]:
+    """
+    Returns records from a source.
+
+    For each record given, call
+    :py:func:`update <dffml.source.source.BaseSourceContext.update>` on the
+    source.
```

Comment:
This still isn't correct, it should say something about how If record keys are given it will yield only those records from the source. If they aren't given then all records in the given source will be yielded.

Reply:
done :)